src/CoreText/CTRunDelegate.cs

```diff
@@ -125,24 +124,10 @@ internal CTRunDelegateCallbacks GetCallbacks ()
 			var callbacks = new CTRunDelegateCallbacks () {
 				version = CTRunDelegateVersion.Version1,
 				dealloc = Deallocate,
+				getAscent = GetAscent,
```

Comment:
I think the GC can collect these delegates early (this is not something this PR changes though).

I guess we could store the callbacks in the CTRunDelegate instance.